linera-service/tests/end_to_end_tests.rs

```diff
@@ -881,6 +883,10 @@ async fn test_wasm_end_to_end_same_wallet_fungible(
     Ok(())
 }
 
+// TODO(#2051): The test `test_wasm_end_to_end_non_fungible::service_grpc` is frequently failing
+// with the error `Error: Could not find application URI: .... after 15 tries`.
+// The whole test is marked as ignored since `tokio::test` needs at least one test.
```

Comment:
Can't we annotate the test with something like
```rust
#[cfg(any(feature = "scylladb", feature = "dynamodb", feature = "kubernetes", feature = "remote_net"))]
```
instead?
(Or maybe you have to nest three `any`s for that.)

Reply:
Yes, that could be a solution. I admit I was a little annoyed by this. I thought that the `tokio::test` should handle the case of zero test matching.